### PR TITLE
Synchronous ROS planning scene methods

### LINKED ISSUE #95: Make planning scene methods synchronous
Currently, all planning scene modifications of the ROS backend (add/remove/append collision objects) are sent via topics, so there's no feedback about whether the operation was completed or not, leading to the proliferation of `time.sleep()` all around and the assumption that after waiting a bit, the operation will be finished.

However, ROS has a service to apply planning scene changes in deltas (diffs), which is probably much better for this case. We should switch the current implementation with a service-based one (which can also be used as blocking calls), to simplify user's code.

### PR DESCRIPTION
**Pull request by beverlylytle**
Closes #95 

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [x] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.rst` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.Configuration`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by beverlylytle** [comment]
Removing the advertising of topics and the initialization of the planner seems to really expose the problem with Twisted as the integration tests consistently fail for commit 066071b.  However, they pass on my machine.  Not sure what to do about this...

**Review by gonzalocasas — COMMENTED** [review]
I know it's still draft, but this is looking good!

**Comment by gonzalocasas** [comment]
In the stack trace, twisted errors out on a file called `_legacy.py` or similar. Could it be that twisted itself has changed and we cannot reproduce because we have slightly older versions of it locally?

**Comment by beverlylytle** [comment]
I use Twisted version 20.3.0 locally, and that's what seems to be appearing in the logs for the integration tests.

**Comment by beverlylytle** [comment]
I just created a new conda environment as is done in the install script (major difference being python 3.8 rather than 3.6) and ran the tests again, and they still pass locally.

**Comment by gonzalocasas** [comment]
I checked what this legacy file is but it's only the logging system,so, unrelated.

One option to try to replicate would be to run the tests in an ubuntu docker container.

**Comment by beverlylytle** [comment]
I've done this in a Debian container and I don't get the same errors.

File: conftest.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def pytest_configure(config):
+    from twisted.internet import selectreactor
+    selectreactor.install()
```
Comment (beverlylytle) [inline conftest.py:3]: The integration tests passed (this time) with this addendum, but locally when I ran the integration tests in a Debian container, they would mostly, but not always fail somewhere in the middle of `robot.py`.  I don't know if this chunk of code exposes, shifts the blame to, or introduces this.

**Review by gonzalocasas — APPROVED** [review]
[LGTM](https://lgtm.lol/i/431)

**Pull request merged**